src/render/mod.rs

```diff
@@ -62,19 +62,29 @@ fn build_shape_pipeline(mut shaders: ResMut<Assets<Shader>>) -> PipelineDescript
         ..PipelineDescriptor::new(ShaderStages {
             vertex: shaders.add(Shader::from_glsl(
                 ShaderStage::Vertex,
-                include_str!("shape.vert"),
+                if cfg!(target_arch = "wasm32") {
```

Comment:
In one of my projects I was using simple macro for selecting proper shader:
```
macro_rules! include_glsl {
    ($fn:expr) => {
        if cfg!(target_arch = "wasm32") {
            include_str!(concat!("v300es/", $fn))
        } else {
            include_str!($fn)
        }
    }
}
```

Reply:
I would probably leave this decision to @Nilirad, personally I don't see a need for a macro when its just one set of shaders.

Reply:
I think there is no need to add a macro, since these are the only two places in the repo where the shaders need to be chosen.